public/app/features/explore/LiveTailButton.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import classNames from 'classnames';
+import { css } from 'emotion';
+import memoizeOne from 'memoize-one';
+import { GrafanaTheme, GrafanaThemeType, useTheme } from '@grafana/ui';
+import tinycolor from 'tinycolor2';
+
+const orangeDark = '#FF780A';
```

Comment:
@dprokop Fumbling with the colors here. Not sure if I should just put them in the theme or not. `FF780A` is used but only in color picker.

Reply:
@aocenas I would suggest using a color from the palette that's the closes one instead of introducing a new one. @Ijin08 any suggestions/?

Reply:
@dprokop how would I do that? I mean should I just export it and import here and use the one I need? There is also the other shade which is new so not sure whether it make sense to reuse just for this one, seems like keeping it as an exception is better for now.